apps/explorer/src/components/orders/BridgeDetailsTable/bridgeDetailsTooltips.tsx

```diff
@@ -7,6 +9,15 @@ export const BridgeDetailsTooltips = {
   bridgingTime: 'Expected time for the bridge operation to complete.',
   maxSlippage: 'The maximum allowed slippage for the bridge in percentage.',
   provider: 'The bridging solution provider.',
-  ownerAddress: 'The account address from which the tokens are bridged.',
+  senderAddress: 'The account address from which the tokens are bridged.',
+  accountProxy: (
+    <span>
+      The{' '}
+      <ExternalLink href="https://swap.cow.fi/#/account-proxy" target="_blank" rel="noopener noreferrer">
```

Comment:
Nitpick, probably for a different PR/task to not delay this change.

Would be nice to point to the appropriated swap env, like it's done the other way around (when linking to explorer from cowswap): https://github.com/cowprotocol/cowswap/blob/develop/libs/common-utils/src/explorer.ts